.kitchen.yml

```diff
@@ -11,6 +11,9 @@ platforms:
   - name: ubuntu-12.04
   - name: debian-6.0.8
   - name: centos-6.4
+    run_list:
+      - recipe[yum-epel::default]
+      - recipe[yum-remi::default]
```

Comment:
Should both of these recipes be included by the cookbook itself if package_install is set true and platform is centos?  Possibly down in the default recipe  where we have the unless package_install block?